docker/render.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ ! -f /config/render.conf ]; then
+    cp /render.conf /config/render.conf
+fi
+
+mapcrafter -c /config/render.conf -j 2
```

Comment:
I would suggest to verify that mapcrafter is not already running before starting a new instance. Some iterations, especially the first one could be long to render.
Something along those line could do the job:
pidof mapcrafter > /dev/null || {
  mapcrafter -c /config/render.conf -j 2
}